src/sentry/api/endpoints/group_details.py

```diff
@@ -144,6 +145,15 @@ def _get_available_issue_plugins(self, request, group):
                                              _with_transaction=False)
         return plugin_issues
 
+    def _get_context_plugins(self, request, group):
+        project = group.project
+        return serialize([
+            plugin
+            for plugin in plugins.configurable_for_project(project, version=None)
```

Comment:
do you actually want `configurable_for_project` here and not `for_project`? to me it seems like all you'd want to check for is whether it's enabled